core/src/services/obs/core.rs

```diff
@@ -180,6 +213,36 @@ impl ObsCore {
         self.send(req).await
     }
 
+    pub fn obs_get_head_object_request(
```

Comment:
Oh, `obs_get_head_object` is wrong, please change into `obs_head_object`.

And `obs_head_object` can call `obs_head_object_request`